.github/workflows/sdk-build.yml

```diff
@@ -32,4 +28,14 @@ jobs:
         flake8 ./src --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with green
       run: |
-        green -vvv ./tests/unit
\ No newline at end of file
+        green -vvv -r ./tests/unit
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
+      with:
+        directory: ./
+        fail_ci_if_error: true
+        files: ./coverage
```

Comment:
@richardgreg A secret token shouldn't be necessary for public repositories. I think the issue is related to the `files` defined here (see the Codecov error below):
```
[2022-06-18T15:[39](https://github.com/zowe/zowe-client-python-sdk/runs/6949083223?check_suite_focus=true#step:7:40):32.958Z] ['error'] None of the following appear to exist as files: ./coverage
[2022-06-18T15:39:32.959Z] ['error'] There was an error running the uploader: Error while cleaning paths. No paths matched existing files!
```

Should the path be `./.coverage` (with an extra `.`) instead of `./coverage`?